daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/SBuiltin.scala

```diff
@@ -1903,6 +1903,9 @@ private[lf] object SBuiltin {
         throw SErrorDamlException(IE.FailedAuthorization(nid, fa))
       case Some(Tx.DuplicateContractKey(key)) =>
         throw SErrorDamlException(IE.DuplicateContractKey(key))
+      case Some(e) =>
```

Comment:
consider matching on the specific error (InconsistentKeys) in this PR even if you fixup the error handling in a separate PR.